site/src/routes/+page.svelte

```diff
@@ -1,10 +1,13 @@
 <script>
+    import { ptLinkFromSlug } from './professors';
+    import { formatClasstime } from './formatting';
+    // import { getCourseLookup, searchCourses } from './courseSearch';
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, I commented it out because it's not currently used in `+page.svelte` but I wanted to show how it could be imported.